2018-November-27/writingSample/rangel.tex

```diff
@@ -0,0 +1,155 @@
+\item
+  \textbf{Problems with this method}
+\item
+  \textbf{Possible new method}
+\end{enumerate}
+
+\textbf{\ldots{}. in the 500 words allowed. I hope that you will take
+this as somewhat of a ``break'' from the more rigorous, and frankly
+better, submissions we have read thus far. }
+
+\textbf{My main concern is how well I was able to communicate the above
+ideas, in a way that comes off as summary, but is also understandable
+and not to scatter-shot. I suppose it could be read like an Introduction
+to a longer paper, so any suggestions on making it feel more like that,
+would be swell!! }
+
+\textbf{I have included figures as a visual aid. }
+
+\textbf{See you all soon!}
+
+\textbf{\emph{Looking towards non-TMS methods of Cortical-Spinal
+Inhibition}}
+
+Transcranial Magnetic Stimulation (TMS) provides relatively focal and
+well tolerated stimulations of neuronal populations, while remaining
+non-invasive. When applied over primary motor cortex (M1), TMS produces
+a corresponding spike in muscular activity known as a Motor Evoked
+Potential (MEP, Figure 1.). If the cortical neurons responsible for the
+movement of that muscle are experiencing inhibition (GABAergic) at the
+time of the TMS pulse, their neurotransmitter release will be shunted,
+and the MEP will have decreased amplitude (Cortico-Spinal Inhibition,
+Figure 2.). TMS remains as one of the primary tools for the measurement
+of inhibition at the cortical level, with applications for pharmacology,
+physical therapy and cognitive studies.
+
+However, TMS presents both experimental issues and unexplained phenomena
+that must be taken into account during research. TMS coil direction
+alone has been hypothesized to preferentially activate different
+populations of M1 interneurons, resulting in different effects on the
+MEP (Figure 3.). Alternate coils (H-Coils), have been devised to reduce
+any pulse-to-pulse variability in position due to experimenter error,
+but they are so far largely restricted to clinical settings.
+Additionally, stands used to hold the TMS coil to the head of the
+participant can also eliminate placement variability, but require
+immobility on the part of the participant, which can limit the
+flexibility of behavioral tasks. Ultimately, even when the TMS coil
+position is inert, MEP amplitudes are known to be vary pulse-to-pulse
+and can be suppressed by the TMS pulses themselves over the course of a
+longer experiment. TMS pulses also produce loud sounds and haptic
+sensations on the skull, which can become distracting and/or startling
+at higher intensities, and activate other, non-target, muscles. Finally,
+patients with epilepsy or traumatic brain injury are often excluded from
+TMS studies as a precaution, which eliminates the prospect of studying
+inhibition in such a unique population. Human studies looking at
+Cortico-Spinal inhibition with TMS must be designed such that they
+accommodate these physiological drawbacks. While TMS still remains a
```

Comment:
would you define all the issues you've raised related to physiology? Or are some mechanical (shape of coil) and practical (restricting movement)